tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -611,6 +611,84 @@ def __call__(self, input1, input2):
 
     return model, model_ref, {}
 
+def shape_of_const_fold_test(temp_dir):
```

Comment:
Why these functions are not added into `class TestMoConvertTF(CommonMOConvertTest)`?

Reply:
I didn't commit the line where they were added to list of tests. Corrected it, thanks!